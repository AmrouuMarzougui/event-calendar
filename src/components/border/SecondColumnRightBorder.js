export const SecondColumnRightBorder = ({ index }) => {
  return (
    <span
      className="border-right"
      style={{
        gridRow: `${index * 12 + 1} / 12 span`,
        gridColumn: "3 / 4",
      }}
    />
  );
};
