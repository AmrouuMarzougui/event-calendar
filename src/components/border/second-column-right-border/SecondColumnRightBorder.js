export const SecondColumnRightBorder = ({ index }) => {
  return (
    <span
      className="border-right"
      data-testid="second-column-right-border"
      style={{
        gridRow: `${index * 12 + 1} / 12 span`,
        gridColumn: "3 / 4",
      }}
    />
  );
};
