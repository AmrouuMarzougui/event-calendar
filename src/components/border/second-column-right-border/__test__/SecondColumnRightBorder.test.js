import React from "react";
import { render, cleanup } from "@testing-library/react";
import "@testing-library/jest-dom/extend-expect";
import renderer from "react-test-renderer";
import { SecondColumnRightBorder } from "../SecondColumnRightBorder";

describe("app component", () => {
  afterEach(cleanup);

  it("render without crushing", () => {
    const { getByTestId } = render(<SecondColumnRightBorder index={5} />);
    expect(getByTestId("second-column-right-border")).toBeTruthy();
  });

  it("it matches snapshots", () => {
    const { tree } = renderer
      .create(<SecondColumnRightBorder index={5} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
