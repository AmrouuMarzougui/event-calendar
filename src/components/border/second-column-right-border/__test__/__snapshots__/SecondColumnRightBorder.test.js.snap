// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app component it matches snapshots 1`] = `undefined`;
